app/components/coin_description_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Coin Description</h2>
+<dl class="dl-horizontal dl-invert coin-details">
+<% fields.each do |field| -%>
+  <%= field %>
```

Comment:
@sandbergja does the data list `<dd> <dt> `fields get output by field?

Reply:
Yes!  fields is a ViewComponent "slot", which is filled with `Blacklight::MetadataFieldComponent`s, which output the `<dd> <dt>` fields.  Here is where the slot is defined: https://github.com/projectblacklight/blacklight/blob/v7.22.2/app/components/blacklight/document_metadata_component.rb#L5